spark/src/main/scala/smile/data/SmileDataFrame.scala

```diff
@@ -0,0 +1,40 @@
+/*******************************************************************************
```

Comment:
I will move this file to smile.spark package as we have more spark related facilities now. It is better to keep them in a separated namespace.

Reply:
ok